ui/src/generated/defaults.js

```diff
@@ -8,5 +8,13 @@ export default {
   ],
   "AMM_NAME": "amm",
   "BRIDGE_URL": "http://127.0.0.1:8000",
-  "CONTRACT_NAME": "VaultFactory"
+  "CONTRACT_NAME": "VaultFactory",
+  "RUN_STAKE_ON_CHAIN_CONFIG": [
+    "getAgoricNames",
+    [
+      "instance",
+      "getRUN"
```

Comment:
now that we're past **agoricdev-8**, this should be...

```suggestion
      "runStake"
```

Reply:
Done, thanks!